CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 * Add `jest` tests.
 * Add `eslint` configuration.
```

Comment:
We have to add namespace fix here too, what do you think ?

Reply:
I did not think it necessary because It's technically implicit in the 0.0.3 changelog.
If you mean the addition to the README, the `Add some documentation in README` covers it.